### DISCUSSION TITLE
Question for the plugin structure. Why seperate the setting of plugin into different file?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1857] [General]
I think make all settings into one file is easier to manage

**Comment by folke** [comment]
you can do whatever you want. one file, or a billion files. It's all the same

**Comment by kobzar** [comment]
> I think make all settings into one file is easier to manage

This is wrong thing.
After a long time you will understand it.
I have a lot of 10 plugins and half of them has 1 screen page length and bellieve me - its hard to make simple changes and jump on the file.
But really - you can add just one file in plugins dir like user.lua and add all of you setting on it.
